tests/dummy/app/templates/index.hbs

```diff
@@ -80,7 +80,7 @@
     </p>
     <p>
       {{!-- Place this tag where you want the button to render. --}}
-      <a class="github-button" href="https://github.com/miguelcobain/ember-paper" data-style="mega" data-count-href="/miguelcobain/ember-paper/network" data-count-api="/repos/miguelcobain/ember-paper#forks_count">Fork</a>
+      <a class="github-button" href="https://github.com/adopted-ember-addons/ember-paper" data-style="mega" data-count-href="/adopted-ember-addons/ember-paper/network" data-count-api="/repos/adopted-ember-addons/ember-paper#forks_count">Fork</a>
```

Comment:
I think these links might need to be changed 🤔 if you rebase this on origin/main you should get a preview for this PR and you can check

Reply:
Yeah, the button API has definitely changed a little! 😂